recipes/pybind11/all/conanfile.py

```diff
@@ -16,17 +16,25 @@ class PyBind11Conan(ConanFile):
 
     _source_subfolder = "source_subfolder"
 
+    _cmake = None
+
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
 
+    def requirements(self):
+        # TODO: self.requires("cpython/x.y/z") or self.requires("pypy/x.y/z")
+        self.output.info("This recipe requires a python SDK which is not available (yet) on CCI")
```

Comment:
@danimtb  LIke this?
```suggestion
        pass
```

Reply:
I'd rather remove the requirements() method completely. We we have something like python I am sure we will revisit recipes that require python somehow

Reply:
Ok. Removed it.